ls1intum/Artemis
# `Development`: Update screenshots and the button location in plagiarism checks docs

### Checklist
#### General

### Motivation and Context
Some of the screenshots in plagiarism checks documentation and path to one button were outdated. The screenshots also contained test data, which should not be presented in the documentation.

### Description
New screenshots are uploaded and new path is typed. 

### Steps for Testing
Read the docs in changed files or [here](https://artemis-platform--7306.org.readthedocs.build/en/7306/user/plagiarism-check.html#plagiarism-check-overview).

#### Code Review
